src/dialog-editor/components/toolbox/toolboxComponent.ts

```diff
@@ -9,7 +10,7 @@ class DialogField {
               options: any = {}) {
     this.icon = icon;
     this.label = label;
-    this.placeholders = Object.assign({
+    this.placeholders = (<any>Object).assign({
```

Comment:
This one sounds wrong .. any way to teach tslint about es6-shim? Or, to assume all ES6 methods are available?